xbmc/GUIInfoManager.cpp

```diff
@@ -3373,6 +3386,30 @@ bool CGUIInfoManager::GetMultiInfoBool(const GUIInfo &info, int contextWindow, c
             bReturn = g_playlistPlayer.GetRepeat(playlistid) == PLAYLIST::REPEAT_ONE;
         }
         break;
+      case LIBRARY_HAS_ROLE:
+      {
+        std::string strRole = m_stringParameters[info.GetData1()];
+        // Find value for role if already stored
+        int artistcount = -1;
+        for (std::vector<std::pair<std::string, int>>::iterator i = m_libraryRoleCounts.begin(); i != m_libraryRoleCounts.end(); ++i)
```

Comment:
needs {} for readability, someone editing this will break it
could also use a for (const auto & role : m_libraryRoleCounts) for readability or at least auto i = m_libraryRoleCounts.begin()